src/main/scala/org/broadinstitute/dsde/firecloud/model/Notification.scala

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.dsde.firecloud.model
+
+/**
+ * Created by mbemis on 6/22/16.
+ */
+sealed trait Notification {
+  val userId: String
+  val notificationId: String
+  def toMap: Map[String, String]
+}
+
+case class ActivationNotification(recipientUserId: String) extends Notification {
+  val userId = recipientUserId
+  val notificationId = "8c646115-d58b-45c2-a8ea-a4f5d47d5d2e"
```

Comment:
What is this UUID? Is this a hardcoded sendGrid ID? Maybe this should be in config.